examples/defaulter-gen/generators/defaulter.go

```diff
@@ -562,8 +558,20 @@ func populateDefaultValue(node *callNode, t *types.Type, tags string, commentLin
 		node.markerOnly = true
 	}
 
-	node.defaultIsPrimitive = t.IsPrimitive()
 	node.defaultType = t.String()
+	if t.Kind == types.Alias {
+		parsedName := types.ParseFullyQualifiedName(t.Name.String())
+		tracker := namer.NewDefaultImportTracker(types.Name{})
+		localPackage := generator.GolangTrackerLocalName(&tracker, t.Name)
+		if len(localPackage) > 0 {
+			node.defaultType = localPackage + "." + parsedName.Name
+		} else {
+			node.defaultType = parsedName.Name
+		}
+		node.underlying = t.Underlying
```

Comment:
Does this work when there is an alias of an alias? I think its worth adding a test for that case and a depth of 3 for good measure.